docs/accessibility/changelog.mdx

```diff
@@ -9,6 +9,10 @@ sidebar_position: 200
 
 # Changelog
 
+## Week of 3/04/2025
+
+- We've upgraded our axe-core version to [4.10.2](https://docs.deque.com/axe-release-impact/4.10.2/en/release-notes)
```

Comment:
```suggestion
- We've upgraded our axe-core version to [4.10.2](https://docs.deque.com/axe-release-impact/4.10.2/en/release-notes).
```

Reply:
We should mention that the axe-core version used to generate a report can be viewed within the Run Properties since this is also new